src/main/java/org/octri/hpoonfhir/controller/MainController.java

```diff
@@ -82,30 +74,22 @@ public String search(Map<String, Object> model, @ModelAttribute PatientModel for
 	}
 
 	/**
-	 * Get the patient observations and convert them to HPO if possible.
+	 * Get the patient information and return to the phenotypes view
 	 * @param model
-	 * @param request
-	 * @return the patient and all associated ObservationModels
+	 * @param id the patient id
+	 * @return the patient found
 	 */
-	@GetMapping("/labs/{id}")
-	public String labs(Map<String, Object> model, @PathVariable String id) {
+	@GetMapping("/patient/{id:.+}")
```

Comment:
Is this ensuring an `id` value exists? It doesn't look like the regex is restricting the possible values.

Reply:
The Epic server has a lots of ids that include a period character which Spring truncates by default. This REGEX allows dots in the path variable: https://stackoverflow.com/questions/16332092/spring-mvc-pathvariable-with-dot-is-getting-truncated

Reply:
Oh gotcha. That's interesting. Good to know.